articles/how-to-upload-files-to-aws-s3-using-django-rest-framework/index.md

```diff
@@ -0,0 +1,437 @@
+# How to upload files to AWS S3 using Django Rest Framework
+
+File upload and download are some most performed actions on the web. With the rise of cloud-based solutions, companies are moving from an on-premises solution to cloud storage. One of the reasons is its cheaper cost and convenience.
+
+This article shows how to upload user-generated files to Amazon S3 using the Django Rest Framework.
+
+## Table of Content
+
+1. [Prerequisites](#prerequisites)
+
+2. [What is AWS S3?](#what-is-AWS-S3?)
+
+3. [Building a simple Django Rest API Application](#building-a-simple-django-rest-api-application)
+
+4. [Integrating AWS S3 into the Django Rest API application](#integrating-aws-s3-into-the-django-api-application)
+
+5. [Summary](#summary)
+
+### Prerequisites
+
+1. Comfortable working with Django and Django Rest Framework
+
+### What is AWS S3?
+
+AWS S3 is an acronym for Amazon Web Services Simple Storage Service (AWS S3). It is a cloud-based service by Amazon for object storage.
+
+Object storage is a type of storage where items processed as a data object. Contrary to the traditional method of storing files in the file system hierarchy.
+
+In traditional file system, the basic unit of storage is a "file". In AWS S3, the basic unit of storage called a "bucket".
```

Comment:
Consider changing "In traditional file system" to "In the traditional file system" and "...storage called a "bucket"." to "...storage is called a "bucket"."